packages/components/forms/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@contentful/f36-forms",
+  "version": "4.0.0",
+  "private": true,
+  "description": "Forma 36: Form components",
+  "scripts": {
+    "build": "parcel build index.ts"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.6.2",
```

Comment:
Should `@babel/runtime` be here?

Reply:
huh! I never noticed this in dependencies! nvm me!

Reply:
@suevalov Do we need this in every package?